tabular/setup.py

```diff
@@ -38,8 +38,9 @@
     "catboost": [
         # CatBoost wheel build is not working correctly on darwin for CatBoost 1.2, so use old version in this case.
         # https://github.com/autogluon/autogluon/pull/3190#issuecomment-1540599280
-        'catboost>=1.1,<1.2 ; sys_platform == "darwin"',
-        "catboost>=1.1,<1.3",
+        # Catboost 1.2 doesn't have wheel for python 3.11
+        "catboost>=1.1,<1.2 ; sys_platform == 'darwin' and python_version < '3.11'",
```

Comment:
they have it by now for mac os 11+, i updated my branch

Reply:
I requested they revert their mac pipeline, no response yet though ref https://github.com/catboost/catboost/issues/2481

Reply:
So we did notice the wheel being available, but for some reason our platform test still tries to build it, which failed